addons/web_editor/static/src/js/editor/odoo-editor/test/spec/list.test.js

```diff
@@ -631,12 +631,12 @@ describe('List', () => {
                             stepFunction: toggleCheckList,
                             contentAfterEdit: unformat(`
                                 <ul class="o_checklist">
-                                    <li class="o_checked" id="checkId-1">abc</li>
+                                    <li class="o_checked">abc</li>
                                     <li class="oe-nested">
                                         <ul class="o_checklist">
-                                            <li class="o_checked" id="checkId-2">def</li>
-                                            <li id="checkId-4">g[]hi</li>
-                                            <li class="o_checked" id="checkId-3">jkl</li>
+                                            <li class="o_checked">def</li>
+                                            <li>g[]hi</li>
+                                            <li class="o_checked">jkl</li>
```

Comment:
I don't understand this commit.

Reply:
This is all an adaptation to the fact that an earlier commit (one that's being forward ported) applies the `removeCheckIds` option to `contentAfterEdit`.
1. The ids in question were named checklist-id-* in 15.0 but are checkId-* in 16.0 so I adapt the code below to that.
2. The option is true in this test so now that it applies to `contentAfterEdit`, the ids are removed.
3. There's no reason to apply the option to `contentAfterEdit` and not to `contentBeforeEdit` so I add that here too.

Since that same code was triplicated, I just edited to put it in a function.